source/process/security.md

```diff
@@ -88,27 +91,166 @@ The following resources are monitoring for information about new security threat
 1. Is there defined management oversight who is responsible for application quality and security reporting & signoff?
    - Yes. 
 
-1. Is access to and maintenance of applications, systems, network components (including routers, databases, firewalls, voice communications servers, voice recording servers, etc), operating systems, virtualization components, hypervisors, or other information objects restricted to authorized personnel only?
+1. For all IT systems including but not limited to servers, routers, switches, firewalls, databases, and external social spaces, is management approval required prior to creating all user and privileged accounts (e.g., system or security administrator)?
    - Yes.
 
-1. Is access to and maintenance of applications, systems, network components (including routers, firewalls, voice communications servers, voice recording servers, voice response units (VRU) etc), operating systems, virtualization components, hypervisors, or other information objects granted based upon need-to-know job function?
+1. For all IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator) logged at all times and reviewed on at least a quarterly basis?
    - Yes.
 
-1. For all IT systems including but not limited to servers, routers, switches, firewalls, databases, and external social spaces, is management approval required prior to creating all user and privileged accounts (e.g., system or security administrator)?
+1. Are all system, application and device password files encrypted using an industry standard encryption algorithm where technically feasible?
+   - Yes
+
+1. Do IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator) that communicate directly with the Internet contain any personally identifiable information (PII) such as: social security numbers, credit card numbers, patient health record information, or other confidential records?
```

Comment:
Would this sentence read better as:

"For all IT systems including but not limited to servers, routers, switches, firewalls and databases, do privileged accounts (e.g., system or security administrator) that communicate directly with the Internet, contain any personally identifiable information (PII) such as: social security numbers, credit card numbers, patient health record information, or other confidential records?"

Reply:
Yes! Updated